test/compiler/parsing/fixtures/watf/nan/payload/actual.wast

```diff
@@ -0,0 +1,3 @@
+(func (
+  f32.const nan:0x400012
```

Comment:
This works because you only have one instructions here.

Could you reformat it to:
```wat
(func (f32.const nan:0x400012))

;; or

(func (
  f32.const nan:0x400012
))
```